### DISCUSSION TITLE
购买后授权设备无法收到验证码，也没有激活码，请帮处理

### DISCUSSION THREAD

**Discussion by LLMIKU** [getlantern/lantern-forum-cn#612] [General]
支付邮箱为angelpoint@foxmail.com
aliPay付费成功后，授权使用专业版： 提示发送了验证码，一周内邮箱无新收件（不在垃圾箱和黑名单）： [angelpoint@foxmail.com]
支付单号src_1PRAnfHGmABNLbfY3qsMJ***（2024.06.13-18:23） 如何联系客服？@Mike777Khoo

**可以发送激活码到我邮箱？或者有没有办法能收到邮件？或者如何退款？**

另外Issues里面好像也有不少遇到一样问题的。
在之前一个帖子里没有回复，就新开了这个帖子。

**Comment by MaxMKMax** [comment]
@LLMIKU https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

**Reply by LLMIKU** [reply]
好的，已经选择第1项无法登入的选项，点击了submit。主要邮箱收不到验证码无法登录，需要授权电脑和手机，之后如果收激活码，能登录2台设备吗
<img width="774" alt="Screen Shot 2024-06-21 at 03 24 57" src="https://github.com/getlantern/lantern-forum-cn/assets/19746677/82514a0c-05aa-4b7a-b314-dbfaa62ffb80">

**Reply by LLMIKU** [reply]
收到激活码的邮件了，成功授权了，非常感谢！可以删除掉这个帖子了
